topics/statistics/tutorials/flexynesis_classification/tutorial.md

```diff
@@ -0,0 +1,749 @@
+>    - *"Flexynesis plot"*: `Dimensionality reduction`
+>        - {% icon param-file %} *"Predicted labels"*: `table` (output of **Sort** {% icon tool %})
+>        - {% icon param-file %} *"Embeddings"*: `job.embeddings_test` (output of **Extract dataset** {% icon tool %})
+>        - *"Column in the labels file to use for coloring the points in the plot"*: `Column: 5`
+>        - *"Transformation method"*: `UMAP`
+>
+{: .hands_on}
+
+# Longer training
+
+In reality, hyperparameter optimization should run for multiple steps so that the parameter search space is large enough to find a good set. However, for demonstration purposes, we only run it for 5 steps here:
+- *"Number of iterations for hyperparameter optimization."*: `5`
+
+> <hands-on-title> Flexynesis </hands-on-title>
+>
+> 1. {% tool [Flexynesis](toolshed.g2.bx.psu.edu/repos/bgruening/flexynesis/flexynesis/0.2.20+galaxy3) %} with the following parameters:
+>    - *"I certify that I am not using this tool for commercial purposes."*: `Yes`
+>    - *"Type of Analysis"*: `Supervised training`
+>        - {% icon param-file %} *"Training clinical data"*: `train_clin_brca.tabular`
+>        - {% icon param-file %} *"Test clinical data"*: `test_clin_brca.tabular`
+>        - {% icon param-file %} *"Training omics data"*: `train_gex_brca.tabular`
+>        - {% icon param-file %} *"Test omics data"*: `test_gex_brca.tabular`
+>        - *"What type of assay is your input?"*: `gex`
+>        - In *"Multiple omics layers?"*:
+>            - {% icon param-repeat %} *"Insert Multiple omics layers?"*
+>                - {% icon param-file %} *"Training omics data"*: `train_cna_brca.tabular`
+>        - {% icon param-file %} *"Test clinical data"*: `test_cna_brca.tabular`
+>                - *"What type of assay is your input?"*: `cna`
+>        - *"Model class"*: `DirectPred`
+>        - *"Column name in the train clinical data to use for predictions, multiple targets are allowed"*: `Column: 16` (CLAUDIN_SUBTYPE)
+>        - In *"Advanced Options"*:
+>            - *"Fusion method"*: `intermediate`
+>            - *"Variance threshold (as percentile) to drop low variance features."*: `0.8`
+>            - *"Minimum number of features to retain after feature selection."*: `100`
+>            - *"Top percentile features (among the features remaining after variance filtering and data cleanup) to retain after feature selection."*: `10.0`
+>            - *"Number of iterations for hyperparameter optimization."*: `5`
+>        - In *"Visualization"*:
+>            - *"Generate embeddings plot?"*: `Yes`
+>            - *"Generate kaplan meier curves plot?"*: `No`
+>            - *"Generate hazard ratio plot?"*: `No`
+>            - *"Generate scatter plot?"*: `No`
+>            - *"Generate concordance heatmap plot?"*: `No`
+>            - *"Generate precision-recall curves plot?"*: `No`
+>            - *"Generate ROC curves plot?"*: `No`
+>            - *"Generate boxplot?"*: `No`
```

Comment:
If you are not changing the default values, for example, *"Generate boxplot?"*: `No`, then there is no need to show them here. It will just make this list longer. Just write those attributes whose values are required to be changed to get the same results as the tutorial. It will make the tutorial shorter in length and look nicer.